### DISCUSSION TITLE
Evaluarea potentialului terenului din evolutia imaginii satelitare in zone similare

### DISCUSSION THREAD

**Discussion by octavflorescu** [ai-in-actiune/idei-proiecte#8] [General]
1. Compararea similaritatii a doua zone. E.g. oras1 (avansat) si oras2 (mai putin avansat, dar similar cu oras1 acum 5 ani)
2. Gasirea zonelor cu potential investitional din orasul 2, ca peste 5 ani sa fie mai valoroase, conform paralelei cu orasul 1.

**Comment by johnsmithm** [comment]
ca set de date, ar trebui preturile la terenuri, este un api de la fisc?

**Reply by octavflorescu** [reply]
Ma gandeam ca un teren de la marginea orasului: acum vs acum 5 ani: daca acum este un bloc pe el, i-a crescut mult valoarea
